dashboard/lib/projects_list.rb

```diff
@@ -42,5 +84,35 @@ def get_project_row_data(student, project, channel_id)
         updatedAt: project_value['updatedAt'],
       }.with_indifferent_access
     end
+
+    def fetch_published_projects_all_types(limit)
+      PUBLISHED_PROJECT_TYPES.map do |type|
+        [type, fetch_published_projects_by_type(type, limit)]
+      end.to_h
```

Comment:
Sounds good. Given that we plan to cache, I'd rather not optimize this into an overly complicated query, especially since we may be joining against other tables soon too (See Scalability section in the PR description).